otelzap/otelzap.go

```diff
@@ -43,8 +43,8 @@ func NewOtelCore(loggerProvider otel.LoggerProvider, opts ...Option) zapcore.Cor
 type Option func(c *otlpCore)
 
 // WithLevel sets the minimum level for the OpenTelemetry Core log to be exported
-func WithLevel(level zapcore.Level) Option {
+func WithLevel(levelEnabler zapcore.LevelEnabler) Option {
```

Comment:
To keep backward compatibility, will it be possible to have another function `WithLevelEnabler` ?

Reply:
Sure! I had figured that since `zapcore.Level` implements `zapcore.LevelEnabler` the change was backwards compatible, but I think you're right that there are scenarios, where it is in fact a breaking change.

I have added `WithLevelEnabler` like you suggested 🙂